outbox/docker-compose.yaml

```diff
@@ -1,5 +1,14 @@
 version: '2.1'
 services:
+  jaeger:
+    image: jaegertracing/all-in-one:1
+    ports:
+     - 5775:5775/udp
+     - 6831:6831/udp
+     - 6832:6832/udp
```

Comment:
In typical cases for Java applications, you should only need the ports `6831/udp` (agent) and `16686/tcp` (UI). You'll need the port `14268/tcp` only if you are exporting data directly from the client (your application) to the collector, bypassing the agent. In that case, you don't need `6831/udp`.

Reply:
@jpechane, can we limit the number of ports?

Reply:
@gunnarmorling I think it is unnecessary. It mimics official jaeger docs https://www.jaegertracing.io/docs/1.21/getting-started/#all-in-one